io-package.json

```diff
@@ -190,21 +190,25 @@
 		"store_quarters" : false,
 		"store_years" : false,
 		"store_period" : false,
-		"keep_days": "7",
```

Comment:
You are using those like numbers, so make them numbers!

Reply:
good one thank you !